Source/WebKit/WPEPlatform/wpe/WPEView.cpp

```diff
@@ -58,6 +59,7 @@ struct _WPEViewPrivate {
         guint button { 0 };
         guint32 time { 0 };
     } lastButtonPress;
+    GRefPtr<WPEGestureController> gestureController { WPE_GESTURE_CONTROLLER(g_object_new(WPE_TYPE_GESTURE_CONTROLLER, nullptr)) };
```

Comment:
This should be initialized on demand, we don't want to create the controller if touch events are never going to be created. We should also use `std::optional` to know whether this is explicitly set to nullptr or not yet initialized